nano/node/rpc.cpp

```diff
@@ -3314,6 +3314,44 @@ void nano::rpc_handler::unchecked_keys ()
 	response_errors ();
 }
 
+void nano::rpc_handler::unopened ()
+{
+	std::unordered_map<nano::account, nano::uint128_t> unopened;
```

Comment:
I guess we need something ordered to receive oredered final result. Also now it's always modifying only last record of set, we don't need to search for random keys

Reply:
The pending iterator guarantees it loops through accounts first? I'll change it

Reply:
Yeah, pending keys ordered by accounts first